### DISCUSSION TITLE
转换cookie为json 文件后导入时间过长无响应

### DISCUSSION THREAD

**Discussion by sinLuoT** [mikumifa/biliTickerBuy#168] [提问专区]
步骤：
扫码登陆，转换好json文件
打开bilibilibuy.exe
手动输入网址http://127.0.0.1:7860/
点击登录管理
点击导入，选择转换好的文件
如图，响应时间过长
![Uploading 无标题.png…]()

**Comment by sinLuoT** [comment]
已经289秒了

**Comment by WittF** [comment]
图片看不到
